src/lib/builders/date-field/_internal/structuredClone.ts

```diff
@@ -0,0 +1,8 @@
+import structuredClonePolyfill from "@ungap/structured-clone";
+
+// Export existing implementation if available.
+if (!("structuredClone" in globalThis)) {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	globalThis.structuredClone = structuredClonePolyfill;
```

Comment:
Rather than overriding global this, I believe having a `clone` method which calls either structuredClone or the polyfill would be best. Unless the issue is coming from an internal library, in which case a PR should be open there instead.